packages/status-js/src/consts/peers.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * source: @see https://fleets.status.im
+ */
+
+// note!: users may experience additional latency due to cross-regional connection
```

Comment:
notice:

Reply:
I don't think we should force anyone to use a particular peer, but that's definitely an option for a package consumer.

Reply:
agreed, just pointing out the current behavior. also, a discovery protocol will be request sooner than any other option for the consumers